api/Animation.json

```diff
@@ -177,6 +177,42 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "endpoint_inclusive_timing": {
+          "__compat": {
+            "description": "`fill: 'forwards'` or `fill: 'both'` not needed",
+            "mdn_url": "https://developer.mozilla.org/docs/Web/API/Animation/commitStyles",
+            "spec_url": "https://drafts.csswg.org/web-animations-1/#dom-animation-commitstyles",
```

Comment:
Spec link and url are the parent. There is no specific section about this topic.